src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -8,7 +8,7 @@
 import com.eppo.sdk.dto.EppoValueType;
 import com.eppo.sdk.dto.ExperimentConfiguration;
 import com.eppo.sdk.dto.Rule;
-import com.eppo.sdk.dto.SubjectAttributes;
+import com.eppo.sdk.dto.EppoAttributes;
```

Comment:
Since we now use the attribute map (it's a `HashMap<String, EppoValue>`) for subjects _and_ actions (assignment options), I renamed it to `EppoAttributes`.